plugin-build/src/test/kotlin/io/sentry/android/gradle/instrumentation/TestSpanAddingParameters.kt

```diff
@@ -0,0 +1,55 @@
+package io.sentry.android.gradle.instrumentation
+
+import java.io.File
+import org.gradle.api.file.RegularFileProperty
+import org.gradle.api.internal.file.DefaultFileCollectionFactory
+import org.gradle.api.internal.file.DefaultFileLookup
+import org.gradle.api.internal.file.DefaultFilePropertyFactory
+import org.gradle.api.internal.file.FileCollectionFactory
+import org.gradle.api.internal.file.collections.DefaultDirectoryFileTreeFactory
+import org.gradle.api.internal.provider.DefaultProperty
+import org.gradle.api.internal.provider.PropertyHost
+import org.gradle.api.internal.tasks.DefaultTaskDependencyFactory
+import org.gradle.api.provider.Property
+import org.gradle.api.tasks.util.internal.PatternSets
+import org.gradle.internal.nativeintegration.services.FileSystems
+import org.gradle.internal.nativeintegration.services.NativeServices
+
+class TestSpanAddingParameters(
+    private val inMemoryDir: File
+) : SpanAddingClassVisitorFactory.SpanAddingParameters {
+
+    init {
+        // TODO: Unresolved reference: initialize
+//        NativeServices.initialize(inMemoryDir)
```

Comment:
@romtsn not sure if its the Gradle version, but `NativeServices.initialize` does not exist